src/components/tabs/__internal__/tabs-header/tabs-header.component.tsx

```diff
@@ -21,21 +32,50 @@ const TabsHeader = ({
   position = "top",
   role,
   extendedLine,
-  alternateStyling,
   noRightBorder = false,
   isInSidebar = false,
 }: TabHeaderProps) => {
+  const [leftScrollOpacity, setLeftScrollOpacity] = useState(0);
+  const [rightScrollOpacity, setRightScrollOpacity] = useState(1);
+
+  const ref = useRef<HTMLDivElement>(null);
+
+  let isScrollable = false;
+
+  const { current } = ref;
+
+  if (position === "top" && current) {
+    isScrollable = current?.scrollWidth > current?.clientWidth;
```

Comment:
```suggestion
    isScrollable = current.scrollWidth > current.clientWidth;
```

conditional chaining isn't necessary since we already know (from the `if`) that `current` exists